src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -147,6 +147,14 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, policyMembers, r
             brickRoadIndicator: !isEmptyObject(reimbursementAccount?.errors) ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
         },
     ];
+    const collectPolicyMenuItems: WorkspaceMenuItem[] = [
```

Comment:
I'm not sure if `collectPolicyMenuItems` is the best name for these menu items since they will also be on control policies. Maybe `moreFeaturesMenuItems` since these pages will all be toggled by the `More features` page?

Reply:
It seems like we're pushing these to `protectedMenuItems` since that already checks for admins of the policy. Let's move it to that list instead of creating a new one.

Reply:
We don't want to use the same list as protectedMenuItems as we want to show only the relevant menu items for Collect policies.

If we decide to reuse the protectedMenuItems, we would show it to Free policies too which is not what we want.

See https://github.com/Expensify/App/pull/36345#discussion_r1492307470 and https://github.com/Expensify/App/issues/36606

Reply:
moreFeaturesMenuItems makes sense to me 👍 if you agree with that

Reply:
I agree that we'd need to conditionally add this to `protectedMenuItems` if we chose to use that list instead.

Reply:
Yes, but if we do this we will have also the Free workspace items along with the Collect ones which not what we want.
I'd say that's not ideal as we want to have a separate nav between free / paid policies.
There's currently a [PR](https://github.com/Expensify/App/pull/36672) that is being reviewed with a specific nav for paid policy.

If we conditionally render it in one big list, it can quickly become a mess as nav items may have a specific order that won't be the same in other workspace types.

Reply:
I think its fine to create two lists: protectedfree and protectedPaid. It will be clean and at the end of a day we are going to remove free very soon